yo1k/sandbox/postgresql/pgexercises/psycopg_basic.py

```diff
@@ -0,0 +1 @@
+# Here will be some useful code.
```

Comment:
What is the reason for including this file in this PR?

Reply:
Done.

There is no real reason for that, it is just my taste but ok, I will not add this dummy file.